script/cargo_cbindgen.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import sys
+import subprocess
+import shutil
+
+from rust_deps_config import RUST_DEPS_PACKAGE_VERSION
+
+
+def get_target_arch(target):
+    return {
+        'arm-linux-androideabi': 'arm',
```

Comment:
can we pass the arch and target in args so we don't have this duplicated here and https://github.com/brave/brave-core/pull/5142/files#diff-39dec1a9847cf2bd0e291145b345cc74R37 ?